sphinx-doc/source/index.md

```diff
@@ -1,14 +1,10 @@
-
 Welcome to the O3R's documentation!
 =============================================
 
 :::{toctree}
 O3R <documentation/O3R/README>
 ifm3d library <ifm3d/doc/sphinx/index>
 ROS packages <ROS/index>
+Downloads <downloadable/index>
```

Comment:
Suggestion: can we use a more “speaking name” instead of Downloads.
Name it something like: documentation history

Reply:
I was trying to keep it open in case we wanted to have other downloadable things, like white papers, or something like that. What do you think @desengph ?

Reply:
Make sense. 
To me, this sounds like a list / table of possible downloads. Sounds reasonable to collect them in place, but I would also reference them in the main documentation:
Example: the VPU / Docker fact sheet can be downloaded here, but can also be accessed from the main Docker documentation chapter.

Reply:
Sounds good to me! We can have the file name as a variable (in conf.py) and reference it at multiple places for instance.